src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,31 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector.getInstance(
+                MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector.getInstance(
+                AuthenticationService.class);
+        try {
+            User registeredUser = authenticationService.register("email@gmail.com", "password");
+            System.out.println(registeredUser);
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Failed to register a new user."
+                    + " Please check your email or password.");
```

Comment:
Consider logging the exception details in the catch block before throwing a new `RuntimeException`. This will help in debugging by providing more context about the failure.